.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 services:
   - rabbitmq
 env:
+  global:
+  - GRADLE_OPTS="-Xmx512m"
```

Comment:
What is the purpose of this change?

Would be great to have itemization in the commit message with the reason of each modification.
At least those which are not obvious.
Thanks